package.json

```diff
@@ -9,18 +9,28 @@
   },
   "scripts": {
     "build": "tsc",
-    "prepublish": "tsc"
+    "prepublish": "tsc",
+    "test": "NODE_ENV=test mocha --exit"
   },
   "dependencies": {
-    "@asteasolutions/zod-to-openapi": "^1.2.1"
+    "@asteasolutions/zod-to-openapi": "^1.2.1",
+    "zod": "^3"
```

Comment:
We might want to move this line to devDeps (so it's a peer dependency for installs, and a dev dependency for the tests etc). I don't totally remember but I think this dev dep can create an odd scenario where you get this:

```
node_modules/zod << one version
node_modules/express-zod-openapi-autogen/node_modules/zod << this version
```

which can cause weird incompatibility between types